src/modules/launcher/Wox.Plugin/IPlugin.cs

```diff
@@ -17,5 +18,11 @@ public interface IPlugin
 
         // Localized description
         string Description { get; }
+
+        /* The two property lines are commented because they brake the unit tests. (The Moq package used in the unit tests doesn't support the .Net 7 feature 'static abstract' properties yet.) - https://github.com/Moq/Moq/issues/1398
```

Comment:
```suggestion
        /* The two property lines are commented because they break the unit tests. (The Moq package used in the unit tests doesn't support the .Net 7 feature 'static abstract' properties yet.) - https://github.com/Moq/Moq/issues/1398
```